spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'menu'
+require 'order'
+
+describe 'User Stories' do
+  let(:menu)  { Menu.new }
+  let(:order) { Order.new(sms) }
+  let(:sms) { double :sms }
+  let(:dish_1) { double :dish, :name => 'steamed pork dumpling', :price => 1.50 }
+  let(:dish_2) { double :dish, :name => 'spare ribs', :price => 4.50 }
+  let(:dish_3) { double :dish, :name => 'prawn crackers', :price => 1 }
+
+  def one_hours_time
+    now = Time.new
+    "%02d" % (now.hour + 1) + ":" + "%02d" % now.min
+  end
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'so I can see the dishes on the menu' do
+    expect { menu.dishes }.not_to raise_error
+  end
+
+  # As a customer
```

Comment:
Try to use 'describe' and 'it' blocks to describe what is under test, rather than having commented user stories. Will make it easier to read the tests, and should help you work in small steps.